templates/openapi3/reference.dot

```diff
@@ -16,12 +16,41 @@ An example how the data schema looks like, take a look at the
 <a href="#response">response object</a>. Additional reading can be done at
 <a href="http://jsonapi.org/" target="_blank" rel="noopener">jsonapi.org</a>.
 
-{{ for (var s in data.components.schemas) { }}
-{{ var origSchema = data.components.schemas[s]; }}
-{{ var schema = data.api.components.schemas[s]; }}
 
-{{= data.tags.section }}
+{{
+var activityItems = [];
+var restItems = [];
+
+for (var s in data.components.schemas) {
+  if (s.startsWith("activity")) {
+    activityItems.push(s);
+  } else {
+    restItems.push(s);
+  }
+}
+}}
+
+{{ var group = { name: "activity", members: activityItems }; }}
```

Comment:
Could you make it so that the 'activity` schema object is the  root object at `reference#activity`? Right now it is located at `reference#activity-activity` which breaks all references for the other activity types.

Reply:
🤣

Reply:
Should be fixed now!